hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AtttributeTypeResolver.kt

```diff
@@ -0,0 +1,57 @@
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import uk.gov.justice.digital.hmpps.prisonersearch.common.model.Prisoner
+import uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch.AttributeType
+import uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch.Attributes
+import kotlin.reflect.KClass
+import kotlin.reflect.KProperty1
+import kotlin.reflect.full.memberProperties
+
+@Configuration
+class AttributeTypeResolver {
+  @Bean
+  fun attributes(): Attributes = getAttributes(Prisoner::class)
+}
```

Comment:
All of the supported attributes are now available in a Bean so they can be used easily by different parts of the system (coming soon).